pwiz_tools/Shared/PanoramaClient/RemoteFileDialog.cs

```diff
@@ -0,0 +1,1668 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json.Linq;
+using System.Windows.Forms;
+
+
+namespace pwiz.PanoramaClient
+{
+    public partial class RemoteFileDialog : Form
```

Comment:
It doesn't look like RemoteFileDialog is being used in the project.  Can it be removed, along with RemoteFileDialog.resx and RemoteFileDialog.Designer.cs?